pkg/app/utils.go

```diff
@@ -47,13 +47,16 @@ func EnsureAppProjectName(userNSClient v1.NamespaceInterface, ownedProjectID, cl
 		}
 	}
 
-	// move Namespace into a project
 	expectedAppProjectName := fmt.Sprintf("%s:%s", clusterName, ownedProjectID)
-	appProjectName := ""
-	if projectName, ok := deployNamespace.Annotations[nslabels.ProjectIDFieldLabel]; ok {
-		appProjectName = projectName
+	appProjectName := deployNamespace.Annotations[nslabels.ProjectIDFieldLabel]
+	if deployNamespace.Name == namespace.System && appProjectName != "" {
```

Comment:
This should use the setting to cover all system-namespaces. This only uses the tenant rancher setting though

Reply:
Done